bookviewer/proj.android-studio/app/debug/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.enuma.booktest"
-    android:sharedUserId="com.enuma.kitkitschool">
+    package="com.maq.xprize.booktest"
+    android:sharedUserId="com.maq.xprize.booktest">
```

Comment:
Should **com.maq.kitkitschool.hindi** come here because that is the _sharedUserId_ you are using across. Am I right?